README.md

```diff
@@ -31,51 +31,89 @@ The docker-compose recipe is set up so that you can use the containers for devel
+with full permissions on that vhost.
+
+Default ports for Rabbit MQ server are `15672` (for accessing the web console) amd `5672` for the amqp protocol used to send messages.
+
+First build the image (the credentials are builtin).
 ```
 docker build -t cami/rabbitmq:1.0 -f docker/cami-rabbitmq/Dockerfile .
 ```
-
-Run a container. We need to specify the hostname since it si usde by rabbitmq
-nodes to identify themselves.
+Run a container. We need to specify the hostname since it is used by rabbitmq nodes to identify themselves.
 ```
 docker run -d --hostname cami-rabbitmq --name cami-rabbitmq -P cami/rabbitmq:1.0
 ```
 
-Get the port on which we can acces the rabbitmq management interface. In this
- case the default port was is
-```
-vagrant@docker:~/cami-project$ docker ps -l
-CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                                                                                                                                                     NAMES
-47101a9ced98        cami/rabbitmq:1.0   "docker-entrypoint.sh"   4 seconds ago       Up 2 seconds        0.0.0.0:32775->4369/tcp, 0.0.0.0:32774->5671/tcp, 0.0.0.0:32773->5672/tcp, 0.0.0.0:32772->15671/tcp, 0.0.0.0:32771->15672/tcp, 0.0.0.0:32770->25672/tcp   cami-rabbitmq
-```
-
-Access the management interface.
+Get the corresponding local port on which we can acces the rabbitmq management interface and the amqp protocol. 
 ```
-http://127.0.0.1:32771
+$ docker ps -l
 ```
+In this case we search for the entries `0.0.0.0:32778->5672/tcp` and `0.0.0.0:32776->15672/tcp`, meaning that:
+- to access the rabbitmq web admin console we use `http://{IP}:32776`
+- the amqp rabbit url will be `amqp://cami:cami@{IP}:32776/cami`  `[2]`
 
 ### cami-medical-compliance
+This instance hosts the medical compliance module that exposes a REST API through Tastypie over Django. It connects to and uses the `cami` database on the `cami-store` instance and also the RabbitMQ instance from cami-rabbitmq.
+
 Build the image with docker:
 ```
 docker build -t cami/medical-compliance:1.0 -f docker/cami-medical-compliance/Dockerfile .
 ```
 
-To run the container you first need to have a running `cami-store` container for the mysql dependency:
-`docker run -d --name cami-store -P cami/store:1.0`
+To run the container you first need to have a running `cami-store` container for the mysql dependency and a running `cami-rabbitmq` container for the mq server (see prev 2 sections).
 
-Then run the `cami-medical-compliance` container linking it to the `cami-store` container.
-``
+For the **development environment**, we need to extract the ports for the mysql-database and for the amqp url of rabbit mq: see `[1]` and `[2]`.
+These should be placed in `cami-project/medical_compliance/medical_compliance/settings.py` where we see the `DEV` tags and **never be commited** in git.
```

Comment:
@paraschivionut you can import a settings_dev, settings_prod, etc. file and put that one in .gitignore. I find this instruction way too complicated to remember :)